test/jdk/java/foreign/TestVarArgs.java

```diff
@@ -190,14 +184,6 @@ private static void check(int index, MemorySegment ptr, List<Arg> args) {
         try (Arena arena = Arena.ofConfined()) {
             MemorySegment seg = ptr.asSlice(0, layout)
                     .reinterpret(arena, null);
-            /* Vararg float is promoted to double (8 bytes) in native (See libVarArgs.c)
-             * in which case float must be converted back from double at the same memory
-             * address in java on the Big-Endian(BE) platforms such as AIX.
-             */
-            if (isAixOS && (layout instanceof ValueLayout) && (((ValueLayout)layout).carrier() == float.class)) {
```

Comment:
Please remove the (now unused) definition of `isAixOS` and other changes relative to upstream.

Reply:
Fixed.